tweets/OpenJSWorld.tweet

```diff
@@ -0,0 +1 @@
+Come hang with us at #OpenJSWorld, the annual global conference hosted by the @openjsf! This free, virtual event is happening on June 2nd and will feature some awesome community and technical talks. Register today: https://openjsf.org/openjs-world-2021/ 
```

Comment:
Rephrasing because this isn't particularly language I'd ever expect from the Node.js account - a few bits of it end up being a bit too close to a fellow kids tone.

 ```suggestion
The @openjsf is holding their free event, #OpenJSWorld, on June 2nd. There'll be number of excellent talks, including a number on Node.js.

If you'd like to attend, you can register here: 
https://openjsf.org/openjs-world-2021/ 
```

Reply:
@RRomoff you resolved this without committing it. You'll need to click "commit suggestion" above to get it to be included in the PR, unless your intent was to reject the suggestion without commenting on why you were doing so.